### PR TITLE
Add may post

### PR DESCRIPTION
**Pull request by d-m-u**

### CODE REVIEW

**Comment by JPrause** [comment]
LGTM :+1: 

@syncrou can you have a look.
@cybette

File: site/_posts/2018-06-04-yet-another-last-month-in-manageiq.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+_It’s an oppressively clammy summer Saturday afternoon. You have the weekend to write YALMiMIQ. It's Saturday and you can afford to procrastinate. You open the Github query and peruse the recent additions anyway, wondering what to showcase this time._
+
+We have an [awesome new virtual column](https://github.com/ManageIQ/manageiq/pull/17509) for ```archived?``` which lets us use it in API queries.
+
+Our [multi-value attribute for group memberships](https://github.com/ManageIQ/manageiq/pull/17497) got some love.
```
Comment (syncrou) [inline site/_posts/2018-06-04-yet-another-last-month-in-manageiq.md]: @d-m-u - Interesting, I clicked this url and [this](https://www.bing.com/images/search?view=detailV2&ccid=mC%2bAVCcw&id=15F9C6DE39116218B49F8D9A8ABD1DBA994404F7&thid=OIP.mC-AVCcwT9zZwJuz5JLZNAHaEo&mediaurl=http%3a%2f%2fimages2.fanpop.com%2fimage%2fphotos%2f9800000%2fTerminator-2-terminator-9844417-1680-1050.jpg&exph=1050&expw=1680&q=terminator+2&simid=608045806695418163&selectedIndex=5&ajaxhist=0) showed up.  I'm thinking that is a surprise?

Comment (d-m-u) [reply site/_posts/2018-06-04-yet-another-last-month-in-manageiq.md]: Funny.

**Comment by JPrause** [comment]
@cybette this blog post is ready for merge. :+1:

File: site/_posts/2018-06-04-yet-another-last-month-in-manageiq.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Last Month in ManageIQ: YALMiMIQ"
+author: d-m-u
+date: 2018-05-01
```
Comment (cybette) [inline site/_posts/2018-06-04-yet-another-last-month-in-manageiq.md]: Maybe a date in June? :)

**Comment by cybette** [comment]
Thanks @d-m-u! just need a little date fix and it'll be good to go :+1:

**Pull request merged**